### PR TITLE
404 page with button

### LINKED ISSUE #31: Implement 404 Page
image

Location: **404Page** (To see the page on localhost just do "localhost:3000/anything")
Number of Images: **2, _404.webp_**

Description: Component includes one images as background. On top of the image should be a button bring the user back to the home page. When hover on the links, the opacity of the button should be lowered. When the user click on the button, the opacity should be lowered more, and it should bring them to the home page. To overlay two tags/components, look into tailwindcss "absolute and relative".  To center the components on the image horizontally/vertically, look into tailwindcss "flex, justify-center, and items-center."

Path: **\ucr-hearts-website\src\app\not-found.tsx**

[Example for implementing a button](https://github.com/acm-ucr/robosub-website/blob/dev/src/components/home/JoinIcon.jsx) Ignore props ( {links, text, ..} ) and put actual links and text. For links, make sure to put double quotes around it. 
[Example for overall formatting](https://github.com/acm-ucr/aag-website/blob/dev/src/components/Title.jsx)

### PR DESCRIPTION
**Pull request by Illey1**
Screenshot (533)

### CODE REVIEW

**Comment by Illey1** [comment]
Shadow isn't there and border is light blue because darker brown color isn't available, neither is lowering opacity. The Font also doesn't match because there was none to apply.

File: src/app/not-found.tsx [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+import Link from "next/link";
+
 const NotFound = () => {
-  return <div>404 Page Not Found</div>;
+  return (
+    <div className="relative h-screen w-full overflow-hidden bg-hearts-beige">
+      <Image
+        src={image1}
+        alt="404 Background"
+        fill
```
Comment (ysh0305) [inline src/app/not-found.tsx]: Make sure to not use pure css like fill and style={...}. Only use tailwindcss. Tailwindcss have "w-full" and "object-contain" which do the same as those.

File: src/app/not-found.tsx [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+        style={{ objectFit: "contain" }}
+        className="pointer-events-none z-10"
+      />
+
+      <div className="absolute inset-0 flex items-end justify-center">
+        <Link
```
Comment (ysh0305) [inline src/app/not-found.tsx:17]: Make sure to add tailwindcss duration to the button. What duration do is it create a better transition when user hover on the button.

**Review by ysh0305 — CHANGES_REQUESTED** [review]
Just minor changes! Everything else looks great!! Good job on completing your first issue!

File: src/app/not-found.tsx (hunk 1 above)
Comment (ysh0305) [reply src/app/not-found.tsx]: Can also add a mt-[5%] on the parent div tag to put the image at the better spot. Not sure if you need it but just an idea :)

**Comment by Illey1** [comment]
Screenshot (544)
Updated page, used gray instead of black for shadow

**Review by qhgill — APPROVED** [review]
looks good!

**Review by ysh0305 — APPROVED** [review]

**Pull request merged**